src/ucp/api/ucp.h

```diff
@@ -338,6 +338,23 @@ typedef struct ucp_params {
 
 /**
  * @ingroup UCP_CONTEXT
+ * @brief Context attributes.
+ *
+ * The structure defines the attributes which characterize
+ * the particular context.
+ */
+typedef struct ucp_context_attr {
```

Comment:
let's also add "field_mask" parameter for compatibility - so user could specify which fields in its context attributes should be returned by this function